common/src/services/send.service.ts

```diff
@@ -19,6 +19,8 @@ import { ErrorResponse } from "../models/response/errorResponse";
 import { SendResponse } from "../models/response/sendResponse";
 import { SendView } from "../models/view/sendView";
 
+export const SEND_KDF_ITERATIONS = 100000;
```

Comment:
This isn't a great place to declare this. I think it's better to keep these default settings together, e.g. in the `KdfType` enum file with the others, or move all of them to a dedicated file.

Reply:
Fair enough, I was a bit split since I didn't like adding it to the KDF enum either as it's more of a feature isolated constant.